pkg/controller/plan/migration.go

```diff
@@ -336,6 +337,41 @@ func (r *Migration) Archive() {
 		return
 	}
 
+	if r.Plan.Provider.Source.Type() == v1beta1.Ova {
```

Comment:
please extract this block to a separate function - sonar cloud has a point, that function becomes more complicated then it should be